conf/nginx.conf

```diff
@@ -20,6 +20,8 @@ location ^~ #LOCATION# {
   add_header X-Robots-Tag none;
   add_header X-Download-Options noopen;
   add_header X-Permitted-Cross-Domain-Policies none;
+  # Add data: to allow /ynhpanel.css to be load due to image on data:base64
+  more_set_headers Content-Security-Policy "default-src data:;";
```

Comment:
In my understanding, CSP rules are applied by the web browser. Nginx is only concerned because we're adding that precise header.
My question was oriented towards what you're saying: allowing only from YunoHost instance. Wouldn't that require the `'self'` directive?